package.json

```diff
@@ -10,20 +10,25 @@
     "start": "lerna run watch --parallel",
     "watch": "yarn start",
     "eas": "packages/eas-cli/bin/run",
-    "lint": "eslint . --ext .ts",
+    "lint": "eslint . --ext .ts --ext .js",
     "release": "lerna version",
     "test": "jest"
   },
   "devDependencies": {
+    "@babel/core": "7.15.0",
     "@expo/oclif-dev-cli": "1.24.0-expo",
     "@types/jest": "27.0.1",
     "eslint": "7.32.0",
     "eslint-config-universe": "7.0.1",
-    "eslint-plugin-graphql": "4.0.0",
     "jest": "27.1.0",
     "jest-watch-typeahead": "0.6.4",
     "lerna": "4.0.0",
     "prettier": "2.3.2",
-    "ts-jest": "27.0.5"
+    "ts-jest": "27.0.5",
+    "typescript": "4.4.2"
```

Comment:
If `typescript` is defined in global `package.json`, does it have to be defined in `eas-cli/package.json` too? Or can it be removed?

Reply:
Good idea